Code/src/main/java/nl/utwente/group10/ui/components/AnchoredConnectionLine.java

```diff
@@ -4,19 +4,13 @@
 import javafx.scene.shape.CubicCurve;
 
 /**
- * This class represent a Line visual object in the UI.
- * Each UI element that uses Line properties should extend
- * from here.
- * 
- * Each Line also stores a startAnchor and an endAnchor to keep track
+ * This is a ConnectionLine that also stores a startAnchor and an endAnchor to keep track
  * of origin points of the Line.
  * 
  * For Lines that connect inputs and outputs of Blocks see Connection.
  */
-public class Line extends CubicCurve {
-
-	/** Control offset for this bezier of this line */
-	public static final double BEZIER_CONTROL_OFFSET_Y = 100f;
+//TODO remove this sub-extension? (Keep just ConnectionLine and Connection)
```

Comment:
Heb het voorlopig zo gelaten, op deze manier is de Connection klasse wat overzichtelijker